pull_alerts.py

```diff
@@ -33,6 +33,20 @@ def pretty_output(self):
             self.notes,
         )
 
+    def _csv_output(self):
+        return u'{},{},{},{},"{}"'.format(
```

Comment:
Can we use python's CSV library to create this row so that escaping is done correctly?